### PR TITLE
feat: full emoji list

### PR DESCRIPTION
**Pull request by kellyjosephprice**
| [![PR App][icn]][demo] | Fix RM-XYZ |
| :--------------------: | :--------: |

## 🧰 Changes

Somehow forgot to include this commit in my last PR.

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: lib/owlmoji.js [hunk 1]
```diff
@@ -1,16 +1,51 @@
...
+    description: 'owlbert',
+    category: 'ReadMe',
+  },
+  {
+    names: ['owlbert-thinking'],
+    tags: ['owlbert thinking'],
+    description: 'owlbert',
+    category: 'ReadMe',
+  },
+];
```
Comment (jennspencer) [inline lib/owlmoji.js:34]: the `tag`s and `description`s here seem a little inconsistent

**Review by erunion — APPROVED** [review]

File: lib/owlmoji.js (hunk 1 above)
Comment (kellyjosephprice) [reply lib/owlmoji.js:34]: Oops. It also doesn't really matter, nobody is consuming those.

Comment (kellyjosephprice) [reply lib/owlmoji.js]: Good catch, fixed it up.

**Review by jennspencer — APPROVED** [review]
:owlbert-celebrate:

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.82.0</b>](/readmeio/markdown/releases/tag/v6.82.0)
<br>

**Pull request merged**